### DISCUSSION TITLE
Out of memory: wasm memory

### DISCUSSION THREAD

**Discussion by samelamin** [trustwallet/wallet-core#3535] [Q&A]
Hi when running locally I sometimes receive the below error

```
RangeError: WebAssembly.Instance(): Out of memory: wasm memory
```

Is there a possible memory leak if I call an api that calls `initWasm` alot?
```
    const core = await initWasm();
```

**Comment by satoshiotomakan** [comment]
Hi @samelamin, as far as I know it's only needed to call `initWasm` once on the application start up
